src/components/flash-messages/_flash-messages.scss

```diff
@@ -0,0 +1,27 @@
+.mint-flash-messages {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 100%;
```

Comment:
IMO positon+left+right+top are specific to symfony/web-client and shouldn't be here. @aju @vergilius WDYT?